topics/discord-bot-list/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: discordbotlist
+topic: discord-bot-list
+github_url: https://github.com/discord
+display_name: Discord Bot Lists
+logo: discord-bot-list.png
+short_description: Websites that list Discord Bots, may have voting, and show important stats about Discord Bots.
+url: https://discord.rovelstars.com

```

Comment:
It seems wrong to link to a specific bot list in the topic description like this, since this is a topic containing many of them and none are "official" or authoritative.

```suggestion
```